drivers/bus/isapnp/isapnp.c

```diff
@@ -677,7 +727,7 @@ IsaPnpCreateReadPortDOResources(
     ResourceList->List[0].InterfaceType = Internal;
     ResourceList->List[0].PartialResourceList.Version = 1;
     ResourceList->List[0].PartialResourceList.Revision = 1;
-    ResourceList->List[0].PartialResourceList.Count = 2;
+    ResourceList->List[0].PartialResourceList.Count = ARRAYSIZE(Ports);
```

Comment:
I know this driver uses a lot of ARRAYSIZE(), but that latter one is a Win32 macro and not an NT one (it should be replaced by RTL_NUMBER_OF() ).